### ISSUE TITLE
config and placeholder issues

### ISSUE THREAD

**Issue by dosapati** [sibiraj-s/ngx-editor#220]
Hi,

 Thank you for this ngx editor. We have been using ngx-editor from last 2 years. Recently we started upgrading and noticed, the config and placeholder properties are removed. 

Not sure whats the intent to remove them? how can I change the placeholder based on the component context to the current component?

can't bind to 'config' since it isn't a known property of 'ngx-editor'.
1. If 'ngx-editor' is an Angular component and it has 'config' input, then verify that it is part of this module.

**Comment by sibiraj-s** [comment]
@dosapati Its not removed. checkout the example how to use plugins https://stackblitz.com/edit/ngx-editor?file=src%2Fapp%2Fapp.module.ts

**Comment by dosapati** [comment]
@sibiraj-s Thank you for your reply.

I have 2 problems now

1. 

`ERROR in app/app.module.ts(380,14): Error during template compile of 'AppModule'
  Function calls are not supported in decorators but 'menu' was called.
`
2. The placeholder is defined at AppModule level, how can I change in one of my App Component.

**Comment by sibiraj-s** [comment]
>The placeholder is defined at AppModule level, how can I change in one of my App Component.

the package v5 is still in alpha. component level placeholder is not yet supported.

>  DEV - ng serve is fine and the upgraded ngx editor is working. When I run PROD build (ng build --prod), the app build is throwing this error. Tried the plugin approach as in the same but no luck, they both ends up with same issue.
ERROR in app/app.module.ts(380,14): Error during template compile of 'AppModule' Function calls are not supported in decorators but 'menu' was called.

I am not sure about the error though. can u post a minimal repo where i can reproduce the issue. Meanwhile can you try this [solution](https://github.com/angular/angular/issues/23609#issuecomment-386876693)

**Comment by dosapati** [comment]
@sibiraj-s : Thank you for your reply.

I tried using the options you suggested angular/angular#23609 (comment), angular/angular#23609 (comment) but none of them worked.

I tried to reproduce the error with a sample app using the latest angular 9 and couldn't re produce the same error on the sample app.

I have a 2-year-old angular application using old configurations and running on Angular 8 as of now. I might migrate code to Angular 9 and all the latest configurations in a month or so. Will update on that.

**Comment by danchaseCTS** [comment]
Hi,

I have a config on a per-editor basis (about 4-5 different editor configs, different pages).. I just upgraded to Angular 10, and it's throwing errors. When I upgraded to the new version of ngx-editor, no errors, but I'm not sure how to deal with the configs, it's saying "Can't bind to 'config' since it isn't a known property of 'ngx-editor'."

Am I stuck with ngx-editor4 by chance? Here's what I'm getting: 

node_modules/ngx-editor/fesm2015/ngx-editor.js:1147:26 - error NG1010: Value at position 3 in the NgModule.imports of NgxEditorModule is not a reference
  Value could not be determined statically.

1147                 imports: [CommonModule, FormsModule, ReactiveFormsModule, PopoverModule.forRoot()],
                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
